README.md

```diff
@@ -628,32 +648,16 @@ A signed e-prescription QR Code in its string form.
 **Response**<br>
 The verification information consists of the following information:
 
-<table>
-  <tr>
-   <td>Valid
-   </td>
-   <td>True if the QR code is correctly signed and not tampered with
-   </td>
-  </tr>
-  <tr>
-   <td>Revoked
-   </td>
-   <td>True if the QR code has been marked as revoked
-   </td>
-  </tr>
-  <tr>
-   <td>Dispensed
-   </td>
-   <td>True if the QR code has been marked as fully dispensed
-   </td>
-  </tr>
-  <tr valign="top">
-   <td>Dispensations
-   </td>
-   <td>If available, an array containing each Medicament with a recorded Dispensation event and a list of those events
-   </td>
-  </tr>
-</table>
+| Field         | Description                                                                                                     |
+|---------------|-----------------------------------------------------------------------------------------------------------------|
+| Valid         | True if the QR code is correctly signed and not tampered with                                                   |
+| Revoked       | True if the QR code has been marked as revoked                                                                  |
+| Dispensed     | True if the QR code has been marked as fully dispensed                                                          |
+| Dispensations | If available, an array containing each Medicament with a recorded Dispensation event and a list of those events |
+| Locked        | True if the e-prescription is currently locked.                                                                 |
+| Locked at     | The time at which the e-prescription was locked.                                                                |
+| Lock reason   | The reason for the e-prescription lock. This is available to authenticated actors only.                         |
+| Unlock reason | The reason for unlocking the e-prescription. This is available to authenticated actors only                     |
```

Comment:
It seems we are missing a few fields from the response:

Reply:
These are the only fields related to lock/unlock.

Reply:
I would like to have a complete list. Regardless of lock/unlock.

Reply:
Added in https://github.com/certifaction/hinsign-cli/pull/15/commits/e132bad5e81fed19fadc6e10b83aacc1bf11c848